superset/security/api.py

```diff
@@ -160,3 +163,33 @@ def guest_token(self) -> Response:
             return self.response_400(message=error.message)
         except ValidationError as error:
             return self.response_400(message=error.messages)
+
+    @expose("/custom_login/", methods=["GET"])
+    @event_logger.log_this
+    @safe
+    @permission_name("read")
+    def custom_login(self) -> Response:
+        token_val = request.headers.get("Authorization")
+
+        headers = {"Authorization": token_val, "accept": "application/json"}
+
+        response = requests.get(
+            f"{current_app.config.get('PLATFORM_RBAC_BASE_URL')}/v1/users/me?expand=organizations.accounts",
```

Comment:
I don't think we need to expand the organizations and accounts. A simple `/me` will be enough to retrieve the email.